### ISSUE TITLE
Fatal Exception: java.lang.NullPointerException - NotificationResponseEvent.java

### ISSUE THREAD

**Issue by kandarp-codingscape** [urbanairship/react-native-module#319]
This issue seems to be happening on Android (including emulator) where user taps to open up a notification, and the app crashes.

Problem seems to be here:
https://github.com/urbanairship/react-native-module/blob/33e538e7fca2f01ba316b45d8312cce9a1d4c3cb/urbanairship-react-native/android/src/main/java/com/urbanairship/reactnative/events/NotificationResponseEvent.java#L72

Just for testing, I changed code to:

```
    @Override
    public boolean isForeground() {
        if (actionButtonInfo != null) {
            return actionButtonInfo.isForeground();
        }

        return false;
    }
```

and there's no crash, and the app is opened. (Not 100% sure about the `return false;` part).

Can you please let me know if there's any workaround to this at the moment?

# Preliminary Info

### What Airship dependencies are you using?
"urbanairship-react-native": "^11.0.0"

# Report

### What unexpected behavior are you seeing?
App crashes.

### What is the expected behavior?
App should open.

### What are the steps to reproduce the unexpected behavior?
Send a message from Airship's dashboard to Android (emulator is ok), and tap received notification to open app.

### Do you have logging for the issue?
```
Fatal Exception: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean com.urbanairship.push.NotificationActionButtonInfo.isForeground()' on a null object reference
       at com.urbanairship.reactnative.events.NotificationResponseEvent.isForeground(NotificationResponseEvent.java:73)
  ```

**Comment by jimmyjacobson** [comment]
This issue was discovered in production via crash reporting on our React Native app.

**Comment by rlepinski** [comment]
@kandarp-codingscape Thanks for the report. Will have a fix ASAP. What you have is basically the fix except you should return true instead of false:

```
    @Override
    public boolean isForeground() {
        if (actionButtonInfo != null) {
            return actionButtonInfo.isForeground();
        }

        return true;
    }
```

**Comment by kandarp-codingscape** [comment]
Thanks! For my own curiosity, what does returning true vs. false do here?

**Comment by rlepinski** [comment]
If the plugin should try to deliver the event though a [headless JS task or not.](https://reactnative.dev/docs/headless-js-android) or not. Its probably not a big deal to deliver the event through a headless js task, but if the response is going to foreground the even its not needed.

**Comment by rlepinski** [comment]
Fixed in 11.0.1. Thanks for reporting the issue.
